build.gradle

```diff
@@ -179,7 +179,7 @@ spotless {
             exclude "**/build/**", "**/build-*/**"
         }
         trimTrailingWhitespace()
-        indentWithSpaces(2)
```

Comment:
Is indentWithSpaces deprecated? Can you explain why this (and some of the other changes) are required in the PR description?

Reply:
Ahh, I already wrote it in the description of the issue, nevermind! Resolving.